gluten-ut/spark33/src/test/scala/org/apache/spark/sql/catalyst/expressions/GlutenCastSuite.scala

```diff
@@ -43,6 +43,56 @@ class GlutenCastSuite extends CastSuite with GlutenTestsTrait {
   UDTRegistration.register(classOf[IExampleBaseType].getName, classOf[ExampleBaseTypeUDT].getName)
   UDTRegistration.register(classOf[IExampleSubType].getName, classOf[ExampleSubTypeUDT].getName)
 
+  test("cast array from IntegerType to StringType") {
```

Comment:
Are these tests designed by yourself? Can we put these new tests into `gluten-ut/test`? Generally, we only put revised Spark test in GlutenXXXSuite which is under spark32/33/34/35. And by putting them into `gluten-ut/test`, all Spark version tests and both backends will be covered.